scripts/run_on_kube.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/python
```

Comment:
Here's the tracking issue which implements this in Kubernetes: https://github.com/kubernetes/community/pull/583